source/packaging-guide/hosted-services/travis-ci.rst

```diff
@@ -52,7 +52,7 @@ Super simple uploading of continuous builds (each push) to GitHub Releases. If t
 Using ``upload.sh``
 ^^^^^^^^^^^^^^^^^^^
 
-The :code:`upload.sh` script in the `uploadtool repository <uploadtool-github>`__ is designed to be called from Travis CI after a successful build. By default, this script will *delete* any pre-existing release tagged with :code:`continuous`, tag the current state with the name :code:`continuous`, create a new release with that name, and upload the specified binaries there. For pull requests, it will upload the binaries to transfer.sh instead and post the resulting download URL to the pull request page on GitHub.
+The :code:`upload.sh` script in the `uploadtool repository <https://github.com/probonopd/uploadtool>`__ is designed to be called from Travis CI after a successful build. By default, this script will *delete* any pre-existing release tagged with :code:`continuous`, tag the current state with the name :code:`continuous`, create a new release with that name, and upload the specified binaries there. For pull requests, it will upload the binaries to transfer.sh instead and post the resulting download URL to the pull request page on GitHub.
```

Comment:
The link is defined in the line below. If it is not working, please fix it there. If there is a reason this is not possible, please tell us why.

Reply:
I don't think the `_uploadtool-github` "variable" worked at all before.
I updated the syntax to follow the one on https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages